src/view/Footer.js

```diff
@@ -0,0 +1,111 @@
+                            </div>{/*---art 2 end--- */}
+
+                            <div className='col-sm'>{/*---art 3--- */}
+                                <p>aerticle3</p>
+                            </div>{/*---art 3 end--- */}
+
+                            <div className='col-sm'>{/*---art 4--- */}
+                                <p>article4</p>
+                            </div>{/*---art 4 end--- */}
+
+                            <div className='col-sm'>{/*---art 5--- */}
+                                <p>article4</p>
+                            </div>{/*---art 5 end--- */}
+
+                            <div className='col-sm'>{/*---art 6--- */}
+                                <p>article4</p>
+                            </div>{/*---art 6 end--- */}
+                        
+                    </div>{/*----row 2 end---*/}
+
+                    <div className='row'>{/*----row 3---*/}
+
+                    <div className='col-sm'>{/*---art 7--- */}
+                                <p>article1</p>
+                            </div>{/*---art 7 end--- */}
+
+                            <div className='col-sm'>{/*---art 8--- */}
+                                <p>aerticle2</p>
+                            </div>{/*---art 8 end--- */}
+
+                            <div className='col-sm'>{/*---art 9--- */}
+                                <p>aerticle3</p>
+                            </div>{/*---art 9 end--- */}
+
+                            <div className='col-sm'>{/*---art 10--- */}
+                                <p>article4</p>
+                            </div>{/*---art 10 end--- */}
+
+                            <div className='col-sm'>{/*---art 11--- */}
+                                <p>article4</p>
+                            </div>{/*---art 11 end--- */}
+
+                            <div className='col-sm'>{/*---art 12--- */}
+                                <p>article4</p>
+                            </div>{/*---art 12 end--- */}
+                        
+                    </div>{/*----row 3 end---*/}
+
+                    <div className='row'>{/*---row 4----- */}
+                           
+                            <div className='col copyright'>{/*----col copyright 1----- */}
+                                    <p>an amazon company</p>
+                                </div>{/*----col copyright 1 end----- */}
+
+                        </div>{/*---row 4 end----- */}
+
+                        <div className='row'>{/*---row 5----- */}
+
+                            <div className='col copyright'>{/*----col copyright 2----- */}
+                                    <p>© 1990-2020 by IMDb.com, Inc.</p>
+                                </div>{/*----col copyright 2 end----- */}
+                            
+                        </div>{/*---row 5 end----- */}
+
+                {/*----container---*/}
+                </div>
+        );
+    }
+};
+export default Footer;
```

Comment:
I think we can remove the unnecessary extra lines and please format the code correctly. Use Prettify or Eslint and set up the "Format on save" option in VSCode

https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode
https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint